interrupterror.go

```diff
@@ -0,0 +1,13 @@
+package miniline
+
+// InterruptedError represents the user having exited the prompt with ^C
+type InterruptedError struct{}
+
+// Error is just the string "Interrupted"
+func (e InterruptedError) Error() string {
+	return "Interrupted"
+}
+
+// ErrInterrupted is a singleton InterruptedError
+var ErrInterrupted error = InterruptedError{}
```

Comment:
I realize this is copied code from the old repo, but wouldn't `errors.New("Interrupted")` suffice here?